game/input_additions.rpy

```diff
@@ -0,0 +1,468 @@
+        text_wo_markers = get_content_wo_markers(self)
+
+        # select_start_pos is None so remove delimiters
+        if self.select_start_pos is None:
+            self.update_text(text_wo_markers, self.editable, check_size = True)
+            return
+
+        # Order positions for string indexing
+        mark1_pos = min(self.select_start_pos, self.select_end_pos)
+        mark2_pos = max(self.select_start_pos, self.select_end_pos)
+
+        # insert delimiters into text
+        text_w_markers = text_wo_markers[:mark1_pos]+"\u231C"+text_wo_markers[mark1_pos:mark2_pos]+"\u231F"+text_wo_markers[mark2_pos:]
+
+        # update text
+        self.update_text(text_w_markers, self.editable, check_size = True)
+
+    def get_content_wo_markers(self):
+        """
+            returns content without delimiters
+        """
+        # If nothing is selected, return empty string
+        if self.select_start_pos is None:
+            return self.content
+
+        # return content without delimiters
+        #NOTE: does not change self.content
+        return self.content.replace(self.select_start_char, '').replace(self.select_end_char, '')
+
+    def remove_selected_markers(self):
+        """
+            removes delimiters from content
+        """
+
+        # If nothing's selected, return
+        if self.select_start_pos is None:
+            return
+
+        # remove delimiters from content
+        self.content = get_content_wo_markers(self)
+
+        # make sure caret isn't outside of text bounds now that we've removed delimiters, essentially 2 characters
+        length = len(self.content)
+        self.caret_pos = min(length, self.select_end_pos)
+
+        # reset selection values
+        self.select_start_pos = None
+        self.select_last_end_pos = None
+
+        # update text
+        self.update_text(self.content, self.editable, check_size = True)
+
+
+    # Add new functions to the Input class
+    setattr(renpy.display.behavior.Input, 'move_selected_left', move_selected_left)
+    setattr(renpy.display.behavior.Input, 'move_selected_right', move_selected_right)
+    setattr(renpy.display.behavior.Input, 'move_selected_home', move_selected_home)
+    setattr(renpy.display.behavior.Input, 'move_selected_end', move_selected_end)
+    setattr(renpy.display.behavior.Input, 'render_selected_markers', render_selected_markers)
+    setattr(renpy.display.behavior.Input, 'remove_selected_markers', remove_selected_markers)
+    setattr(renpy.display.behavior.Input, 'move_selected_all', move_selected_all)
+    setattr(renpy.display.behavior.Input, 'selected_copy', selected_copy)
+    setattr(renpy.display.behavior.Input, 'selected_cut', selected_cut)
+    setattr(renpy.display.behavior.Input, 'input_paste', input_paste)
+    setattr(renpy.display.behavior.Input, 'get_selected', get_selected)
+    setattr(renpy.display.behavior.Input, 'remove_selected', remove_selected)
+
+    map_event = renpy.display.behavior.map_event
```

Comment:
Can you add a comment indicating what this does here? Coming at a glance, would make sense to re-add a little bit of docs to help people understand what the function does